packages/plugin-support/src/support.js

```diff
@@ -0,0 +1,139 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+import {defaults} from 'lodash';
+import uuid from 'uuid';
+
+
+const Support = SparkPlugin.extend({
+  namespace: `Support`,
+
+  _constructFileMetadata(metadata) {
+    let metadataArray = [];
+    let client = this.spark.client || {};
+    let trackingId = client.trackingIdBase;
+
+    if (this.spark.config.trackingIdPrefix) {
+      trackingId = `${this.spark.config.trackingIdPrefix}_${client.trackingIdBase}`;
+    }
+
+    [`locusId`, `callStart`, `feedbackId`].forEach(function addKeyToArray(key) {
+      if (metadata[key]) {
+        metadataArray.push({
+          key: key,
+          value: metadata[key]
+        });
+      }
+    });
+
+    metadataArray.push({
+      key: `trackingId`,
+      value: trackingId
+    });
+
+    return metadataArray;
+  },
+
+  getFeedbackUrl(options) {
+    options = options || {};
+    return this.request({
+      method: `POST`,
+      api: `conversation`,
+      resource: `users/deskFeedbackUrl`,
+      body: defaults(options, {
+        appVersion: this.config.appVersion,
```

Comment:
Add and document these values in config.js (even if you leave them as undefined.